packages/core/src/tracing/instrumentation/database/elasticsearchModern.js

```diff
@@ -436,13 +438,13 @@ function forEachApiAction(ESAPI, fn) {
   forEachKeyRecursive(esApiFromBuildApi, [], fn);
 }
 
-function forEachKeyRecursive(obj, path, fn) {
+function forEachKeyRecursive(obj, apiPath, fn) {
```

Comment:
This needed to be renamed because we now have the identifier `path` in the outer scope already (because we require the `path` module`).